README.md

```diff
@@ -240,8 +248,9 @@ $ python manage.py migrate
 ```
 
 
-Start the server:
+## Start the server:
 
+Start the Python server with the `runserver` command.
```

Comment:
nitpick but I'd say "start the Django development server with"

Reply:
Updated.